src/listeners/browser.js

```diff
@@ -70,6 +78,16 @@ export default class BrowserSignalListener {
     }
   }
 
+  _flushImpressionsCount() {
+    const pf = fromImpressionsCountCollector(this.impressionsCounter);
```

Comment:
Name should be something with payload to match the others ideally. it should represent it is the same payload that we send to the other endpoint.